src/WebJobs.Script/Description/CSharp/FunctionEntryPointAttribute.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    [System.AttributeUsage(AttributeTargets.Method, Inherited = false, AllowMultiple = false)]
+    public sealed class FunctionEntryPointAttribute : Attribute
```

Comment:
This attribute is used for entry point (target method/function) resolution. If there are multiple methods and, for any reason, the user wants a method named something other than "Run" to be executed, this attribute may ne used.